### ISSUE TITLE
webrtc not working when chrome udpate to 87 , kms 6.7.1 

### ISSUE THREAD

**Issue by JerryGwd** [Kurento/bugtracker#562]
## Prerequisites

* [] I agree to fill this issue template.
* [] I have read the [Troubleshooting Guide] and [Support Instructions].

[Troubleshooting Guide]: https://doc-kurento.readthedocs.io/en/latest/user/troubleshooting.html
[Support Instructions]: https://github.com/Kurento/.github/blob/master/SUPPORT.md

## Issue description
Webrtc not working, when chrome udpate to 87 verson ,  and there was no ERROR or WARNING in  web with kms 6.7.1 
kms error:
2020-12-08 19:48:40,892866 44750 [0x00007f9a21ffb700]   fixme basesink                  gstbasesink.c:3125 gst_base_sink_default_event() <nicesink161>  stream-start event without group-id. Consider implementing group-id handling in the upstream elements
2020-12-08 19:48:40,893292 44750 [0x00007f9a20ff9700]   fixme default                   gstutils.c:3766 gst_pad_create_stream_id_internal() <nicesrc161:src>  Creating random stream-id, consider implementing a deterministic way of creating a stream-id
2020-12-08 19:48:40,894177 44750 [0x00007f9ab186c700]    info kmswebrtcsession          kmswebrtcsession.c:1529 kms_webrtc_session_start_transport_send() <kmswebrtcsession161>  Started transport for media: audio, index: 0
2020-12-08 19:48:40,894273 44750 [0x00007f9ab186c700]    info kmswebrtcsession          kmswebrtcsession.c:1529 kms_webrtc_session_start_transport_send() <kmswebrtcsession161>  Started transport for media: video, index: 1
2020-12-08 19:48:40,902731 44750 [0x00007f9ab106b700]    info kmswebrtcsession          kmswebrtcsession.c:795 kms_webrtc_session_gather_candidates() <kmswebrtcsession161>  Gather candidates
2020-12-08 19:48:41,910750 44750 [0x00007f9a20ff9700] warning dtlsconnection            gstdtlsconnection.c:694 openssl_poll() <GstDtlsConnection@0x7f9a1c01b080>  do_handshake encountered BIO error
2020-12-08 19:48:43,887349 44750 [0x00007f9a57bdb700] warning dtlsconnection            gstdtlsconnection.c:694 openssl_poll() <GstDtlsConnection@0x7f9a1c01b080>  do_handshake encountered BIO error
2020-12-08 19:48:47,887503 44750 [0x00007f9a57bdb700] warning dtlsconnection            gstdtlsconnection.c:694 openssl_poll() <GstDtlsConnection@0x7f9a1c01b080>  do_handshake encountered BIO error

## Context
We use kms 6.7.1 for long time ,

### INFO about Kurento Media Server

* Kurento version:  
* Server OS:          
* Installation method:
    
  - [] [apt-get]
  - [] [Docker]
  - [] [AWS CloudFormation]
  - [] [Built from sources]

[apt-get]: https://doc-kurento.readthedocs.io/en/latest/user/installation.html#installation-local
[Docker]: https://doc-kurento.readthedocs.io/en/latest/user/installation.html#installation-docker
[AWS CloudFormation]: https://doc-kurento.readthedocs.io/en/latest/user/installation.html#installation-aws
[Built from sources]: https://doc-kurento.readthedocs.io/en/latest/dev/dev_guide.html#dev-sources

### INFO about your Application Server

* Programming Language:  
* Kurento Client version:     

### INFO about end-user clients

* Device(s):    
* OS(es):        
* Browser(s):  

### Run these commands

```
cat /etc/lsb-release
```
DISTRIB_ID=Ubuntu
DISTRIB_RELEASE=16.04
DISTRIB_CODENAME=xenial
DISTRIB_DESCRIPTION="Ubuntu 16.04.6 LTS"

```
kurento-media-server --version
```
Kurento Media Server version: 6.7.1
Found modules:
	'core' version 6.7.1
	'elements' version 6.7.1
	'filters' version 6.7.1

```
dpkg -l | grep -Pi 'kurento|kms-|gst.*1.5|nice'
```
ii  gstreamer1.5-libav:amd64                          1.8.2.1.xenial~20170725171352.96.493eee4   amd64        libav plugin for GStreamer
ii  gstreamer1.5-nice:amd64                           0.1.13.1.xenial~20170725160546.81.eebfdab  amd64        ICE library (GStreamer plugin)
ii  gstreamer1.5-plugins-bad:amd64                    1.8.1.1.xenial~20170725164047.100.3db37b1  amd64        GStreamer plugins from the "bad" set
ii  gstreamer1.5-plugins-base:amd64                   1.8.1.1.xenial~20170725154709.55.7b19cfd   amd64        GStreamer plugins from the "base" set
ii  gstreamer1.5-plugins-good:amd64                   1.8.1.1.xenial~20170725161537.112.9ee4248  amd64        GStreamer plugins from the "good" set
ii  gstreamer1.5-plugins-ugly:amd64                   1.8.1.1.xenial~20170725170621.89.2685b0f   amd64        GStreamer plugins from the "ugly" set
ii  gstreamer1.5-pulseaudio:amd64                     1.8.1.1.xenial~20170725161537.112.9ee4248  amd64        GStreamer plugin for PulseAudio
ii  gstreamer1.5-x:amd64                              1.8.1.1.xenial~20170725154709.55.7b19cfd   amd64        GStreamer plugins for X11 and Pango
ii  kms-core                                          6.7.1.xenial.20180321183643.47e0370        amd64        Kurento Core module
ii  kms-elements                                      6.7.1.xenial.20180321184406.5aaf4a0        amd64        Kurento Elements module
ii  kms-filters                                       6.7.1.xenial.20180321185049.7f2d045        amd64        Kurento Filters module
ii  kms-jsonrpc                                       6.7.1.xenial~20180321183209.1.637bf2a      amd64        Kurento JSON-RPC library
ii  kmsjsoncpp                                        1.6.3.xenial.20170725151047.d78deb7        amd64        Kurento jsoncpp library
ii  kurento-media-server                              6.7.1.xenial.20180321185711.dd49a2c        amd64        Kurento Media Server
ii  libgstreamer-plugins-bad1.5-0:amd64               1.8.1.1.xenial~20170725164047.100.3db37b1  amd64        GStreamer development files for libraries from the "bad" set
ii  libgstreamer-plugins-base1.5-0:amd64              1.8.1.1.xenial~20170725154709.55.7b19cfd   amd64        GStreamer libraries from the "base" set
ii  libgstreamer1.5-0:amd64                           1.8.1.1.xenial~20170725152356.170.0d6031b  amd64        Core GStreamer libraries and elements
ii  libnice10:amd64                                   0.1.13.1.xenial~20170725160546.81.eebfdab  amd64        ICE library (shared library)
ii  openh264-gst-plugins-bad-1.5:amd64                1.8.1.1.xenial~20170725164047.100.3db37b1  amd64        GStreamer plugins from openh264

**Comment by j1elo** [comment]
If you need to use Kurento 6.7, you should use an equivalent version of the browsers (from the same approximate date). You are mixing a very old version of Kurento, with a very new version of Chrome, it is normal that there are problems.

**Comment by julicsantana** [comment]
I'm facing the same problem. I use Kurento Media Server version 6.12.0 and after the chrome update my application stopped working. It works fine on firefox and chrome under version 87. I've checked the log and I'm also getting the do_handshake encountered BIO error.

2020-12-16T09:31:41,817485 9587 0x00007f49b1992700    info webrtcendpoint            kmswebrtcendpoint.c:439 kms_webrtc_endpoint_add_ice_candidate() <kmswebrtcendpoint94>  Add ICE candidate 'candidate:38$
2020-12-16T09:31:42,140661 9587 0x00007f4984b76700 warning dtlsconnection            gstdtlsconnection.c:694 openssl_poll() <GstDtlsConnection@0x7f49a008b4e0>  do_handshake encountered BIO error
2020-12-16T09:31:42,206027 9587 0x00007f4982b72700    info webrtcendpoint            kmswebrtcendpoint.c:278 new_selected_pair_full() <kmswebrtcendpoint95>  New candidate pair selected, local: 'candidate$
2020-12-16T09:31:42,376493 9587 0x00007f497fb6c700    info webrtcendpoint            kmswebrtcendpoint.c:278 new_selected_pair_full() <kmswebrtcendpoint93>  New candidate pair selected, local: 'candidate$
2020-12-16T09:31:42,781838 9587 0x00007f4982b72700 warning dtlsconnection            gstdtlsconnection.c:694 openssl_poll() <GstDtlsConnection@0x55a473e5c6f0>  do_handshake encountered BIO error
2020-12-16T09:31:42,792318 9587 0x00007f497fb6c700 warning dtlsconnection            gstdtlsconnection.c:694 openssl_poll() <GstDtlsConnection@0x7f49a008cd60>  do_handshake encountered BIO error
2020-12-16T09:31:44,100076 9587 0x00007f497db68700 warning dtlsconnection            gstdtlsconnection.c:694 openssl_poll() <GstDtlsConnection@0x7f49a008b4e0>  do_handshake encountered BIO error
2020-12-16T09:31:44,746432 9587 0x00007f497db68700 warning dtlsconnection            gstdtlsconnection.c:694 openssl_poll() <GstDtlsConnection@0x55a473e5c6f0>  do_handshake encountered BIO error
2020-12-16T09:31:44,756216 9587 0x00007f497d367700 warning dtlsconnection            gstdtlsconnection.c:694 openssl_poll() <GstDtlsConnection@0x7f49a008cd60>  do_handshake encountered BIO error
2020-12-16T09:31:48,103549 9587 0x00007f497db68700 warning dtlsconnection            gstdtlsconnection.c:694 openssl_poll() <GstDtlsConnection@0x7f49a008b4e0>  do_handshake encountered BIO error
2020-12-16T09:31:48,747214 9587 0x00007f497db68700 warning dtlsconnection            gstdtlsconnection.c:694 openssl_poll() <GstDtlsConnection@0x55a473e5c6f0>  do_handshake encountered BIO error
2020-12-16T09:31:48,756430 9587 0x00007f497d367700 warning dtlsconnection            gstdtlsconnection.c:694 openssl_poll() <GstDtlsConnection@0x7f49a008cd60>  do_handshake encountered BIO error
2020-12-16T09:31:51,845983 9587 0x00007f497db68700 warning dtlsconnection            gstdtlsconnection.c:694 openssl_poll() <GstDtlsConnection@0x7f49a40b76a0>  do_handshake encountered BIO error

**Comment by j1elo** [comment]
Please test with the Kurento Hello World tutorial to verify a minimal
test case, and to see if Chrome 87 is really breaking Kurento.

https://doc-kurento.readthedocs.io/en/latest/user/tutorials.html#hello-world

I tested earlier today with Kurento 6.15 and the Java Hello World
tutorial, and Chrome Version 87.0.4280.88. It worked without issues.

**Comment by prlanzarin** [comment]
That's Chrome 87 dropping support for SSL/DTLS downgrading. You're using a KMS version old enough that still didn't correctly shipped with DTLS 1.2 enabled, hence the error.

I'd consider this not an issue.

**Comment by j1elo** [comment]
So this issue is a bug that had been already solved in Kurento. This is why the **Step Number 1** of BOTH the [Troubleshooting Guide](https://doc-kurento.readthedocs.io/en/latest/user/troubleshooting.html) and [Support Instructions](https://github.com/Kurento/.github/blob/master/SUPPORT.md) is to test with the **latest version** of Kurento Media Server.

And that's why the Issue Prerequisites ask to read those documents.
